libs/portal-integration-angular/src/lib/core/components/portal-header/header.component.html

```diff
@@ -54,7 +54,7 @@
           <ng-container *ngFor="let item of moreMenuItems">
             <li class="layout-topbar-item" *ocxIfPermission="item.permission">
               <!-- *ngIf="!item?.disabled" -->
-              <a
+              <!-- <a
```

Comment:
Why?

Reply:
forgot to revert, because I get the error: Type 'MouseEvent' has no properties in common with type 'MenuItemCommandEvent' that I couldn't resolve